cookbooks/bach_krb5/recipes/keytabs.rb

```diff
@@ -0,0 +1,42 @@
+get_all_nodes().each do |h|
+  # Create directories to store keytabs
```

Comment:
This will create multiple directories one for each host as keytabs for each host are stored in host specific directory.